code/__DEFINES/dcs/signals/signals_client.dm

```diff
@@ -0,0 +1,4 @@
+/// from base of [/client/proc/handle_spam_prevention] (message, mute_type)
+#define SIGNAL_CLIENT_AUTOMUTE_CHECK "client_automute_check"
```

Comment:
I wasn't clear, sorry--keep this to COMSIG, renaming them all to SIGNAL is something I want to do at some point but should probably be done all at once.

Reply:
Alright, done.